scss/mixins/_progress.scss

```diff
@@ -9,6 +9,11 @@
     background-color: $color;
   }
 
+  // IE10+, Microsoft Edge
+  &[value] {
```

Comment:
also, looking at `scss/_progress.scss` we currently have

```
.progress[value] {
  // IE10 uses `color` to set the bar background-color
  color: #0074d9;
  ...
```
so it feels a bit more consistent to still stick with non-`::-ms-fill` one here. having said all that, i don't feel particularly strongly about it here either way...